source/MapPanel.cpp

```diff
@@ -102,6 +105,16 @@ namespace {
 	const int HOVER_TIME = 60;
 	// Length in frames of the recentering animation.
 	const int RECENTER_TIME = 20;
+
+	bool HasMultipleLandablePlanets(const System *system)
+	{
+		const auto &stellarObjects = system->Objects();
+
+		const auto &isRegularPlanet = [] (const StellarObject &stellarObject) {
```

Comment:
`const` because I always use that for variables that aren't written to. `&` because without that it always looks to me like it does a copy, but I'll drop it.

Reply:
This code got replaced.